mct_quantizers/__init__.py

```diff
@@ -26,4 +26,6 @@
 from mct_quantizers.keras import quantizers as keras_quantizers
 from mct_quantizers.pytorch import quantizers as pytorch_quantizers
 
+from mct_quantizers.pytorch.onnxruntime_session_options import get_ort_session_options
+
 __version__ = "1.2.0"
```

Comment:
test

Reply:
It is tested in the tests since all tests are using it to check inference is possible